src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -35,24 +37,44 @@ export class AddBulkConcepts extends Component {
     this.state = {
       conceptIds: '',
       openResultModal: false,
+      otherSelected: false,
     };
     this.invalidConceptIds = [];
-    this.sourceUrl = 'orgs/CIEL/sources/CIEL/concepts/';
+    this.sourceUrl = 'orgs/CIEL/sources/CIEL/';
```

Comment:
What has this got to do with other sources other than CIEL?

Reply:
CIEL is the default when the page is loaded. `this.sourceUrl = 'orgs/CIEL/sources/CIEL/'` holds the sourceUrl, and it changes when a different source is selected.